THIRD-PARTY-NOTICES

```diff
@@ -0,0 +1,64 @@
+------------------------------------------------------------------------
```

Comment:
Use the format found here:

https://github.com/mongodb/mongo-python-driver/blob/master/THIRD-PARTY-NOTICES#L1-L16

As specified here:

https://jira.mongodb.org/browse/DRIVERS-323

Including specifying what files the license applies to.